### DISCUSSION TITLE
essay/2024/10/29/irreplaceable

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#241] [Comments]
# essay/2024/10/29/irreplaceable

월급이 적다는 이유로 회사를 그만둔 적은 없었지만… 그런 이유를 대봤으면 어땠을까 상상해 봤습니다.

https://jeho.page/essay/2024/10/29/irreplaceable.html

**Comment by yoonjong-park** [comment]
진실.

**Reply by BenjaminKim** [reply]
슬픈 진실 입니다. 😂
